### DISCUSSION TITLE
Using a github action to automate changelog generation

### DISCUSSION THREAD

**Discussion by abhiabhi94** [Radi85/Comment#187] [General]
Hey @Radi85, what do you think about using a GitHub workflow to automate the changelog generation. I saw this: https://github.com/heinrichreimer/action-github-changelog-generator a couple of weeks ago and looks good to me.

It essentially generates `changelog` on the basis of the issues and pull-requests on GitHub. In our case, we can use it any of the two scenarios:
- Generate `changelog` after every pull request is merged. 
    - Pro - Keeps the repository up to date with the latest changes even from the end user point of view, even for unreleased versions.
    - Cons - Maybe pollutes the git history.

- Generate `changelog` after a tag is pushed to the default branch.
    - Pro and cons can be seen as the reverse of the above listed option.

**Comment by SALIMA534** [comment]
Hi , I have integrated django-comments-dab package into my project. Features of django-comments-dab package is but How to use user as our project register as not as admin user it is considering as admin user how to override this and I want to show user as my project registered users. Please help me to achieve this, Thanks.

**Reply by HelloMukama** [reply]
it already does this. From my end it shows the user as the logged in user not admin.
if u login with admin, itll show admin and someone else otherwise
